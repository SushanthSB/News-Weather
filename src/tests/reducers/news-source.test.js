import renderer from 'react-test-renderer';
import NewsSource from '../../reducers/news-source';

it("Array of sources should't alter",() => {
	expect(NewsSource()).toMatchSnapshot()
})