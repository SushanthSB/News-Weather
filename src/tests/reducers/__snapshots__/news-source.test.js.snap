// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Array of sources should't alter 1`] = `
Array [
  "BBC News",
  "The Hindu",
  "The Times of India",
  "The Wall Street Journal",
  "BBC Sport",
  "MTV News",
  "IGN",
]
`;
